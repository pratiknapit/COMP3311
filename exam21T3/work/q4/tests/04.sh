psql property -c 'select id, address(id) from properties where id between 46400 and 46405 order by id;'
