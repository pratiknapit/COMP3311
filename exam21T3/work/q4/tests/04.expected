  id   |                address                
-------+---------------------------------------
 46400 | 72 Spring Street, Bondi Junction 2022
 46401 | 319 Bronte Road, Bondi Junction 2022
 46402 | 12/199 Anzac Parade, Kingsford 2032
 46403 | 7/16 Glenwood Avenue, Coogee 2034
 46404 | 10/84 Bowden Street, Alexandria 2015
 46405 | 19/64 Byron Street, Coogee 2034
(6 rows)
